packages/components/descriptions/src/description-item.ts

```diff
@@ -37,3 +39,9 @@ export default defineComponent({
     },
   },
 })
+export default DescriptionItem
+
+export type DescriptionItemVNode = VNode & {
+  children: SetupContext['slots']
+  props: InstanceType<typeof DescriptionItem>['$props']
+}
```

Comment:
Use `buildProps` to define props and extract prop types?

Reply:
Done